### DISCUSSION TITLE
Crashreport, Crash when i in outpost.

### DISCUSSION THREAD

**Discussion by Dujodan** [FakeFishGames/Barotrauma#13664] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [x] My issue happened while using mods.

### What happened?

So, started new company at the today morning, played ~10hours and when i commonly arrived on new outpost "Wolstone". And everytime after ~5 mins of being inside the outpost i got a crash. This problem repeated about 4 times.

### Reproduction steps

idk, i suppose its can be problem with modpack, but idk why this problem started after long time. 

2 crashreports: 

[2.txt](https://github.com/FakeFishGames/Barotrauma/files/14733149/2.txt)
[3.txt](https://github.com/FakeFishGames/Barotrauma/files/14733150/3.txt)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.2.8.0 (Winter Update hotfix 2)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
Uploaded 2 crashreports to previous box.
```

**Comment by SomeRandomNoobKekeke** [comment]
Это скорее всего из-за нейротравмы, попробуй ее отключить
https://github.com/FakeFishGames/Barotrauma/issues/13450

**Comment by Dujodan** [comment]
> Это скорее всего из-за нейротравмы, попробуй ее отключить #13450

Только что еще поиграл и понял что это происходит когда я собираю кровь у нпс на аванпосту, только почему вдруг эта процедура стала выдавать краш спустя 10 часов геймлпея? Есть ли мб идеи как это фиксить?

**Comment by SomeRandomNoobKekeke** [comment]
Попробуй [вот это](https://steamcommunity.com/sharedfiles/filedetails/?id=3190189044) вместо нейротравмы поставить

**Comment by Dujodan** [comment]
> Попробуй [вот это](https://steamcommunity.com/sharedfiles/filedetails/?id=3190189044) вместо нейротравмы поставить

Спасибо, вроде как помогло.

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/issues/11973
